src/spec.c

```diff
@@ -2792,8 +2810,11 @@ void Indexes_RdbSave(RedisModuleIO *rdb, int when) {
   while ((entry = dictNext(iter))) {
     StrongRef spec_ref = dictGetRef(entry);
     IndexSpec *sp = StrongRef_Get(spec_ref);
+    if (!sp) {
+      continue;
+    }
```

Comment:
Why is this needed?
I think this can never happen, as this is done from the main thread